Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -816,7 +816,7 @@ void MainWindow::loadSettings()
         resetDefaultSoundsPack();
     }
     slotUpdateUI();
-    if (ttSettings->value(SETTINGS_DISPLAY_START_SERVERLIST, SETTINGS_DISPLAY_START_SERVERLIST_DEFAULT).toBool() == true && ttSettings->value(SETTINGS_CONNECTION_AUTOCONNECT, SETTINGS_CONNECTION_AUTOCONNECT_DEFAULT).toBool() == false)
+    if (ttSettings->value(SETTINGS_DISPLAY_START_SERVERLIST, SETTINGS_DISPLAY_START_SERVERLIST_DEFAULT).toBool() == true && ttSettings->value(SETTINGS_CONNECTION_AUTOCONNECT, SETTINGS_CONNECTION_AUTOCONNECT_DEFAULT).toBool() == false && !TT_GetFlags(ttInst) & CLIENT_CONNECTION)

```

Comment:
What evaluates first? "!", "&" or "&&" ? Use parenthesis to ensure it evaluates correctly.